### ISSUE TITLE
Not a valid ES Module

### ISSUE THREAD

**Issue by kenjonespizza** [everflow-io/everflow-sdk#12]
Hi 👋🏾

I'm attempting to use the Everflow SDK in my SvelteKit project, which runs on Vite, but upon installing the `@everflow/everflow-sdk` and running my project, I get the following error.  Is this something you all can update to make your package more usable?

```
@everflow/everflow-sdk doesn't appear to be written in CJS, but also doesn't appear to be a valid ES module (i.e. it doesn't have "type": "module" or an .mjs extension for the entry point). Please contact the package author to fix.
/node_modules/.pnpm/@everflow+everflow-sdk@1.0.8/node_modules/@everflow/everflow-sdk/src/fp_sdk.js:17
                __vite_ssr_import_0__.default.get((components) => {
                                              ^

TypeError: Cannot read properties of undefined (reading 'get')
    at eval (/node_modules/.pnpm/@everflow+everflow-sdk@1.0.8/node_modules/@everflow/everflow-sdk/src/fp_sdk.js:17:47)
    at new Promise (<anonymous>)
    at eval (/node_modules/.pnpm/@everflow+everflow-sdk@1.0.8/node_modules/@everflow/everflow-sdk/src/fp_sdk.js:16:20)
```

Thanks!

**Comment by olivierboucher** [comment]
Please refer to https://github.com/everflow-io/everflow-sdk/issues/6

**Comment by kenjonespizza** [comment]
> TLDR: we do not offer support for the SDK outside the standard usage:

What is the standard usage?  I am building a website using a JavaScript framework.  Is that not standard usage for an NPM package?

**Comment by olivierboucher** [comment]
The standard usage is to load the sdk source from your tracking domain:

```
<script type="text/javascript" 
        src="https://www.YOUR-TRACKING-DOMAIN.com/scripts/sdk/everflow.js"></script>
```

The npm package **works** it just doesn't work with whatever build tools you're using and we're not keen on working to support it. You can always submit a pull request and we'll look at it

**Comment by olivierboucher** [comment]
I also updated the README with sample usage
